pr_agent/settings/configuration.toml

```diff
@@ -230,6 +230,7 @@ push_commands = [
     "/describe",
     "/review --pr_reviewer.num_code_suggestions=0",
 ]
+skip_draft_mr = false
```

Comment:
remove this. it's a dead configuration. should be 'true' always
(and putting something by default as 'False' is like saying that no one will ever use it. We do want to use it)

Reply:
I don't know what do you mean by "dead configuration".

This line serves to:

1. Let the users know that configuration exists and that the default is false. Otherwise where are documented ? 
2. If we set it to true it means introducing a breaking change as now draft requests are handled.

Reply:
dead configuration means something so exotic that 99.9% of PR-agent users won't ever change.
And in addition, here you have (for a 'dead configuration' a wrong default value.

A change is not just for you. It is for the entire community. And its usage should be from the viewpoint of the entire community, and the entire code repo.

Since the correct behaviour is to avoid draft PRs (this is already done, without config, in github) we should do it also in Gitlab.

 if possible and logical, the best configuration is no configuration.

Reply:
So, to be sure to have understood it well, are you proposing to always skip draft MR without having any kind of configuration toggle ? 

In my opinion there could be use cases where you want it enabled and dropping it entirely it seems a bit drastic.

Reply:
@mrT23 anyway, just committed a version without any configuration toggle, we skip it always when the MR is in draft.